docs/audience.md

```diff
@@ -0,0 +1,133 @@
+
+- Project Manager on the stakeholder side
+- Business Analyst on the stakeholder side
+- Business Owner for SMBs
+- Department Lead
+- Digital Transformation Leader
+- External Digital Transformation Consultant
+
+Examples of activities:
+
+- Change the to, from, subject or body of an email generated by the system
+- Add or remove properties in the [Entities](/docs/Entities/Overview.md)
+- Adjust permissions for the users of the application
+- Build [Dashboards](/docs/Dashboards/Overview.md)
+
+## `Citizen Developers`
+
+In Plant an App, Citizen Developers do, on average, 80% of the application development. They live at the intersection of the business and technical realms. On one hand, they work to understand the business requirements and the processes involved. At the same time, they have the ability to turn them into a working software that has a data model, workflows and application screens.
+
+Note that a Citizen Developer rarely has experience with writing code. But they are technical savvy in the sense that they understand technology. The know concepts such as what HTTP Requests are, what is the difference between back-end and front-end technologies or how databases are structured. Often, they even know the basics of various markup and programming languages, such as HTML, CSS, SQL or JavaScript and can read such code for the purpose of copy-pasting and adjusting it.  
+
+Skills: `no-code`, `tech savvy`, `data models`, `business process modeling`, `basic coding skills`
+
+Typical personas:
+
+- Technical Business Analysts
+- Fresh graduates from technical universities
+- Graduates from related online courses
+- Junior Developers
+
+Examples of activities:
+
+- Define the data model through the [Entities](/docs/Entities/overview.md)
+- Implemented the permission scheme
+- Create application screens with forms, grids, dashboards and other visualizations
+- Implement business processes through [Workflows](/docs/Workflows/overview.md) and configure most of the [Actions](/docs/Actions/overview.md)
+- Integrate with external systems by using the [API Builder](/docs/ApiBuilder/oveview.md) and the [Server Request](/docs/Actions/server-request.md) action
+- Build [Dashboards](/docs/Dashboards/Overview.md)
+  
+## `Low-code Engineers`
+
+Plant an App is designed to provide extreme flexibility by allowing custom code to be plugged directly in the application. For example, complex SQL queries can be run using the [Execute SQL](/docs/Actions/execute-sql.md) action or C# code by using the [Execute Razor](/docs/Actions/execute-razor.md) action, just to name a few.
+
+Low-code Engineers step in to write the custom code. Therefore, they have prior coding experience. But, at the same time, they understand low-code and Plant an App because the code that is plugged in is likely to be connected to other things in the [Context](/docs/Workflows/context.md), such as tokens produces by other actions, input parameters into the workflow or the data layer.
```

Comment:
```suggestion
Low-code Engineers step in to write the custom code. Therefore, they have prior coding experience. But, at the same time, they understand low-code and Plant an App because the code that is plugged in is likely to be connected to other things in the [Context](/docs/Workflows/context.md), such as tokens produced by other actions, input parameters into the workflow or the data layer.
```